plaid-stl/src/plaid/mod.rs

```diff
@@ -156,3 +158,14 @@ pub fn get_response() -> Result<String, PlaidFunctionError> {
         Err(_) => Err(PlaidFunctionError::ParametersNotUtf8),
     }
 }
+
+/// Give the runtime more context about an error encountered during execution
+pub fn set_error_context(context: impl Display) {
```

Comment:
In this case, I would have this take an &str to prevent allocations when calling this function